### DISCUSSION TITLE
Creator of tag does not show correct user in tags page

### DISCUSSION THREAD

**Discussion by korydraughn** [community/community#89925] [Repositories] [answered]
So, I created a tag like so ...
```
git tag the_tag the_commit
git push upstream the_tag
```

The tag appeared in the repo, but when I view the tag details, GitHub shows a different user's name.

Is this a bug or did I miss a step?

**Comment by korydraughn** [comment, marked as answer]
Using `git tag the_tag the_commit -am 'message'` fixed the issue.

**Reply by filipeandre** [reply]
Explanation:
-a: This flag creates an annotated tag, which includes a message and stores additional information (such as the author and date).

the_commit can be omitted and it will default to ther last commit
